x-pack/plugins/alerting/server/routes/bulk_edit_rules.ts

```diff
@@ -34,6 +38,27 @@ const operationsSchema = schema.arrayOf(
       field: schema.literal('actions'),
       value: schema.arrayOf(ruleActionSchema),
     }),
+    schema.object({
```

Comment:
Currently functional tests are skipped due to https://github.com/elastic/kibana/issues/132195
I plan to fix them and add additional tests in the following PR, so won't mix everything together.

I can do it in this PR, if it preferable

Reply:
For reviewers: 
I believe this is the follow up PR @vitaliidm is referring to 😉 
https://github.com/elastic/kibana/pull/133635/files